commit ae725e3f9cb4e1eb825ebe1d55241c98c2ea32f1
Author: Tulio Magno Quites Machado Filho <tuliom@linux.ibm.com>
Date:   Mon Jun 15 11:15:57 2020 -0300

    powerpc: Add new hwcap values
    
    Linux commit ID ee988c11acf6f9464b7b44e9a091bf6afb3b3a49 reserved 2 new
    bits in AT_HWCAP2:
     - PPC_FEATURE2_ARCH_3_1 indicates the availability of the POWER ISA
       3.1;
     - PPC_FEATURE2_MMA indicates the availability of the Matrix-Multiply
       Assist facility.

diff --git a/sysdeps/powerpc/bits/hwcap.h b/sysdeps/powerpc/bits/hwcap.h
index b35f5eddc1d309bb..f2853a11df16f63c 100644
--- a/sysdeps/powerpc/bits/hwcap.h
+++ b/sysdeps/powerpc/bits/hwcap.h
@@ -74,3 +74,5 @@
 #define PPC_FEATURE2_SCV	   0x00100000 /* scv syscall.  */
 #define PPC_FEATURE2_HTM_NO_SUSPEND  0x00080000 /* TM without suspended
 						   state.  */
+#define PPC_FEATURE2_ARCH_3_1	   0x00040000 /* ISA 3.1.  */
+#define PPC_FEATURE2_MMA	   0x00020000 /* Matrix-Multiply Assist.  */
diff --git a/sysdeps/powerpc/dl-procinfo.c b/sysdeps/powerpc/dl-procinfo.c
index 35cac2e249916507..4555d4548554e788 100644
--- a/sysdeps/powerpc/dl-procinfo.c
+++ b/sysdeps/powerpc/dl-procinfo.c
@@ -77,7 +77,7 @@ PROCINFO_CLASS const char _dl_powerpc_cap_flags[64][15]
     "", "", "", "",
     "", "", "", "",
     "", "", "", "",
-    "", "", "", "htm-no-suspend",
+    "", "mma", "arch_3_1", "htm-no-suspend",
     "scv", "darn", "ieee128", "arch_3_00",
     "htm-nosc", "vcrypto", "tar", "isel",
     "ebb", "dscr", "htm", "arch_2_07",
